perfkitbenchmarker/providers/openstack/flags.py

```diff
@@ -44,10 +44,28 @@
                     'defaults to $NOVA_ENDPOINT_TYPE.')
 
 flags.DEFINE_string('openstack_public_network', None,
-                    'Name of OpenStack public network')
+                    '(DEPRECATED: Use openstack_floating_ip_pool) '
+                    'Name of OpenStack public network.')
 
 flags.DEFINE_string('openstack_private_network', 'private',
-                    'Name of OpenStack private network')
+                    '(DEPRECATED: Use openstack_network) '
+                    'Name of OpenStack private network.')
+
+flags.DEFINE_string('openstack_network', 'private',
+                    'Name of OpenStack network. Network provides '
```

Comment:
Currently, it needs to be an existing network. The same for the floating IP pool.

On public clouds, this would be the equivalent of your internal network, except it can be named anything which is why we take it as a parameter.

Defaults to 'private' since a lot of people keep that the default name.